src/components/persons.js

```diff
@@ -3625,3 +3625,21 @@ export const persons = [
   },
 ];
```

Comment:
You need to remove this bracket, it's breaking your code. :)

Reply:
Good Luck! :)

Reply:
removed brackets, can you please check now.